modules/etcd-defrag.adoc

```diff
@@ -6,10 +6,43 @@
 [id="etcd-defrag_{context}"]
 = Defragmenting etcd data
```

Comment:
This is probably not what you signed on to do with this module, but if you have time, this could probably broken into two modules:

1. Concept: etcd defragmentation types
2. Procedure: Manual defrags